src/panels/lovelace/common/handle-action.ts

```diff
@@ -76,14 +76,16 @@ export const handleAction = (
         forwardHaptic("light");
       }
       break;
-    case "call-service": {
+    case "call-service":
       if (!actionConfig.service) {
         forwardHaptic("failure");
         return;
       }
       const [domain, service] = actionConfig.service.split(".", 2);
       hass.callService(domain, service, actionConfig.service_data);
       forwardHaptic("light");
-    }
+      break;
+    default:
+      fireEvent(node, actionConfig.action, actionConfig);
```

Comment:
@bramkragten suggest in chat that we should perhaps restrict this to action types prefixed with "custom-"